modules/ROOT/pages/clauses/limit.adoc

```diff
@@ -145,3 +145,29 @@ Writes `locked` property on one node and return that node:
 d|Rows: 1 +
 Properties set: 1
 |===
+
+[role=label--new-5.24]
+[[limit-standalone-clause]]
+== Using `LIMIT` as a standalone clause
+
+`LIMIT` can be used as a standalone clause, or in conjunction with `ORDER BY` or `SKIP`/`OFFSET`.
+
+.Query
+[source, cypher, indent=0]
+----
+MATCH (n)
+ORDER BY n.name DESC
+SKIP 2
+LIMIT 2
+RETURN collect(n.name) AS names
```

Comment:
This query isn't really using it standalone though right? Would standalone not look more like: MATCH (n) LIMIT 2 RETURN collect(n.name) AS names?

Reply:
I think there could be a better way to word it, or possibly have to examples. One with only limit "standalone" and one example of them together as in the vast majority of cases it doesn't make sense to use it by itself without at least and order by.

Reply:
So, one could argue that a separate page or section would be more useful to show the existence of `Order by n.name SKIP 1 LIMIT 1` as a standalone clause